src/main/java/io/supertokens/webserver/api/core/ActiveUsersCountAPI.java

```diff
@@ -52,9 +52,8 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IO
         }
 
         try {
-            int count = ActiveUsers.countUsersActiveSince(
-                    this.getAppIdentifier(req),
-                    this.enforcePublicTenantAndGetPublicTenantStorage(req), sinceTimestamp);
+            enforcePublicTenantAndGetPublicTenantStorage(req); // to enforce this API is called from public tenant
```

Comment:
this is not needed cause etAppIdentifier should check if the app is valid or not as well

Reply:
this is required to enforce public tenant, not for ensuring app is valid